drink/src/runtime/mod.rs

```diff
@@ -4,17 +4,19 @@
 pub mod minimal;
 pub mod pallet_contracts_debugging;
 
-use frame_support::sp_runtime::{AccountId32, Storage};
+use frame_support::sp_runtime::Storage;
+use frame_system::pallet_prelude::BlockNumberFor;
 pub use minimal::MinimalRuntime;
 
+/// The type of an account identifier.
+pub type AccountId<R> = <R as frame_system::Config>::AccountId;
```

Comment:
for consistency you can rename it to `AccountIdFor<R>`

Reply:
+1